test/e2e/framework/helpers/yunikorn/rest_api_utils.go

```diff
@@ -224,6 +224,17 @@ func isRootSched(policy string) wait.ConditionFunc {
 	}
 }
 
+func GetHealthCheck() (dao.SchedulerHealthDAOInfo, error) {
+	restClient := RClient{}
```

Comment:
The rest calls are generally designed as methods on the type RClient. See [GetApps()](https://github.com/apache/incubator-yunikorn-k8shim/blob/4d28705e4723c27178c7f8d31751ecea2150398d/test/e2e/framework/helpers/yunikorn/rest_api_utils.go#L126), or most other methods in the rest_api_utils.go for an example